recruiter/views.py

```diff
@@ -55,3 +54,22 @@ def form_valid(self, form):
         form.instance.company = logged_recruiter.company
         form.instance.date_created = date.today()
         return super(CreateNewJobForm, self).form_valid(form)
+
+
+def recruiterRegister(request):
+    if request.method == 'POST':
+        form = RecuiterRegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            message = "Your account has been created. You can log in now!"
+            messages.success(request, f'{message}')
```

Comment:
Was it really necessary to use `messages` here? It seems to me that since you have dedicated view for presenting the success message, this could have been a simple static string in the template.

This seems to add unnecessary complexity to the code, not to mention that it needs to be tested. Are you sure the messages are preserved across the redirection call? Redirection essentially terminates the HTTP call while sending some redirection headers to the browser causing it to essentially make a brand new HTTP call to another page. Things do not necessarily get preserved across different HTTP calls.

I'm not saying this does not work, you probably at least tested this manually but it does something non trivial that:

1. Forces reviewers to look it up, since in order to work it would need to employ some non-trivial tricks like server-based sessions and cookies behind the scenes, and its not safe to just assume that the `messages` module really does all that.
2. Must be tested via automated tests.

Finally, since I looked further down the code, I see that the `recruiter_created_successfully` template does not event attempt to display the messages, so what we essentially have here is a piece of unnecessary code that sends the reviewers into a wild goose chase. Not nice!